pipelines/ner_wikiner/project.yml

```diff
@@ -31,7 +31,7 @@ commands:
     # caching works correctly.
     script:
       - "python scripts/partition.py assets/aij-wikiner-en-wp2.bz2 assets/iob"
-      - "python -m spacy convert assets/iob corpus --n-sents 10"
+      - "python -m spacy convert assets/iob corpus --converter iob --n-sents 10"
```

Comment:
I think the bug related to this was fixed in `master` and this change isn't necessary?

Reply:
I just felt like it wouldn't hurt to be specific here. But yes, in theory the bug should be fixed on `master` as well!